05_stringer/jessodri/main_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("127.0.0.1\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("actual: %s does not match expected: %s", actual, expected)
+	}
+}
+func TestStringMethod(t *testing.T) {
+	testCases := []struct {
+		description string
+		ip          IPAddr
+		expected    string
+	}{
+		{"happy path", IPAddr{127, 0, 0, 1}, "127.0.0.1"},
+		{"one value", IPAddr{127}, "127.0.0.0"},
+		{"two value", IPAddr{127, 0}, "127.0.0.0"},
+		{"three value", IPAddr{127, 0, 0}, "127.0.0.0"},
+		{"empty", IPAddr{}, "0.0.0.0"},
+	}
```

Comment:
I included a max test too with 255.255.255.255 but it's not necessary

Reply:
I agree with @pfannerj, can you please put a test for the max case `255.255.255.255` this ensures the implementation is done correctly using `byte`